src/constants/routers.ts

```diff
@@ -3,4 +3,15 @@ export const ROUTESTRING = {
   RECORDING: 'recording',
   NOTIFICATION: 'notification',
   PROFILE: 'profile',
+  LOGIN: 'login',
+  SIGNUP: 'signup',
+};
+
+export const ROUTEURL = {
+  SUBSCRIPTION: '/' + ROUTESTRING.SUBSCRIPTION,
+  RECORDING: '/' + ROUTESTRING.RECORDING,
+  NOTIFICATION: '/' + ROUTESTRING.NOTIFICATION,
+  PROFILE: '/' + ROUTESTRING.PROFILE,
+  LOGIN: '/' + ROUTESTRING.LOGIN,
+  SIGNUP: '/' + ROUTESTRING.SIGNUP,
```

Comment:
```suggestion
  SUBSCRIPTION: '/' + ROUTE_STRING.SUBSCRIPTION,
  RECORDING: '/' + ROUTE_STRING.RECORDING,
  NOTIFICATION: '/' + ROUTE_STRING.NOTIFICATION,
  PROFILE: '/' + ROUTE_STRING.PROFILE,
  LOGIN: '/' + ROUTE_STRING.LOGIN,
  SIGNUP: '/' + ROUTE_STRING.SIGNUP,
```
이런식으로 구분자를 두는게 더 가독성이 좋지 않을까요?